<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>topWidget</class>
 <widget class="QWidget" name="topWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>85</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="textField"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="redButton">
       <property name="text">
        <string>Red</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="blueButton">
       <property name="text">
        <string>Blue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="yellowButton">
       <property name="text">
        <string>Yellow</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
